scripts/build/build.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
+
+# shellcheck disable=SC2086
```

Comment:
I had to disable enforcing double quotes around variable expansion, since it resulted in a bug of several flags being passed as one to build.go. build-all.sh already does this, I can't understand how build.sh ever worked with the aforementioned bug.

Reply:
That's worrying. Good find.